### DISCUSSION TITLE
Change in default cache folder location

### DISCUSSION THREAD

**Discussion by vishwa-ims** [RevealBi/Reveal.Sdk#24] [ASP.NET Core] [answered]
Hi,

We are using ASP Web SDK1.4 and wanted to change default location of cache folder which is currently "C:\Temp\Reveal\Web\Cache"

**Comment by TorreyBetts** [comment, marked as answer]
Hello!

If you check out RevealEmbedSettings, you'll find two properties that can change the location; CachePath, and DataCachePath.

Here's the link to the API documentation: [https://help.revealbi.io/api/aspnet/latest/Reveal.Sdk.RevealEmbedSettings.html](https://help.revealbi.io/api/aspnet/latest/Reveal.Sdk.RevealEmbedSettings.html)
